demisto_sdk/commands/content_graph/parsers/base_script.py

```diff
@@ -41,10 +42,15 @@ def field_mapping(self):
                 "type": "type",
                 "subtype": "subtype",
                 "alt_docker_images": "alt_dockerimages",
+                "outputs": "outputs",
             }
         )
         return super().field_mapping
 
+    @property
+    def outputs(self) -> Optional[List]:
```

Comment:
```suggestion
    def outputs(self) -> List[Output]:
```

according to the code below, looks like it can be only a List?